### ISSUE TITLE
Doesn't let me block more than one chain at a time

### ISSUE THREAD

**Issue by amaranth21** [satsukitv/twitter-block-chain#77]
I can run the extension once but then it logs me out of my account and even with other accounts, all I get are error messages. Even when running it on small accounts, I get the same problem.

**Comment by Blockq** [comment]
I just got it today and am seeing the same thing.  Most of the users I want to block return errors.  Out of every thousand or so perhaps 200 get through.  Then it logs me out of my account and I have to log back in, then close Chrome and reopen it to continue.  It is a GREAT tool and very much needed right now but definitely needs tweaking.

**Comment by deviantollam** [comment]
yeah it appears to run up against some kind of Twitter rate limiting.  500 block in rapid succession seems to be a hard trigger, but there are other cumulative ones, etc.

it's a great solution, but may need some pacing and slowdown in the script.
